sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -144,7 +145,8 @@ Ext.define('SlateAdmin.controller.People', {
                 click: me.onSearchClick
             },
             'people-navpanel #groups': {
-                select: me.onGroupSelect
+                //select: me.onGroupSelect
+                itemclick: me.onGroupSelect
```

Comment:
@chris I switched to itemclick because the the root node was initially selected so there was no way to click on it to load all people if there are no groups created yet.   root node is selected on view load and on search clear... don't know if we would want to remove that.  Add logic to make sure there are existing groups before selecting root node?